src/main/resources/modules/wellness_encounters.json

```diff
@@ -455,9 +454,115 @@
           "transition": "Record_EGFR"
         },
         {
-          "transition": "Wellness_Encounter"
+          "transition": "Smoker_Status_Observation"
+        }
+      ]
+    },
+    "Smoker_Status_Observation": {
+      "type": "Simple",
+      "conditional_transition": [
+        {
+          "transition": "Record_Current_Smoker",
+          "condition": {
+            "condition_type": "Attribute",
+            "attribute": "smoker",
+            "operator": "==",
+            "value": true
+          }
+        },
+        {
+          "transition": "Record_Former_Smoker",
+          "condition": {
+            "condition_type": "And",
+            "conditions": [
+              {
+                "condition_type": "Attribute",
```

Comment:
Transition options are processed in order and the first one that evaluates to true gets selected, sort of like an `if` in code, so it's not necessary to explicitly specify "smoker == false" in this condition, or "smoker == false and quit_smoking_age is nil" in the third transition below.